packages/zipkin-transport-aws-sqs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "zipkin-transport-aws-sqs",
+  "version": "0.19.2",
+  "description": "Transports Zipkin trace data via AWS SQS to the collector",
+  "main": "lib/index.js",
+  "types": "index.d.ts",
+  "scripts": {
+    "build": "babel src -d lib",
+    "test": "mocha --exit --require ../../test/helper.js",
+    "test-debug": "mocha --inspect-brk --exit --require ../../test/helper.js",
+    "prepublish": "npm run build"
+  },
+  "license": "Apache-2.0",
+  "repository": "https://github.com/openzipkin/zipkin-js",
+  "dependencies": {
+    "aws-sdk": "^2.7.0",
+    "zipkin": "^0.19.2",
+    "zipkin-encoder-thrift": "^0.19.2"
```

Comment:
I am curious about the need of thrift. I would just go with V2 but would like to @adriancole's opinion.